src/main/flight/navigation_rewrite_multicopter.c

```diff
@@ -481,13 +481,22 @@ bool isMulticopterLandingDetected(uint32_t * landingTimer, bool * hasHadSomeVelo
     // check if we are moving horizontally
     bool horizontalMovement = sqrtf(sq(posControl.actualState.vel.V.X) + sq(posControl.actualState.vel.V.Y)) > 100.0f;
 
-    // Throttle should be low enough
+    // We have likely landed if throttle is 40 units below average descend throttle
     // We use rcCommandAdjustedThrottle to keep track of NAV corrected throttle (isLandingDetected is executed
     // from processRx() and rcCommand at that moment holds rc input, not adjusted values from NAV core)
-    bool minimalThrust = rcCommandAdjustedThrottle < posControl.navConfig->mc_min_fly_throttle;
-    
+    *landingThrSamples += 1;
+    *landingThrSum += rcCommandAdjustedThrottle;
+    bool minimalThrust= rcCommandAdjustedThrottle < *landingThrSum / *landingThrSamples - 40;
```

Comment:
I compared my and stronnags bblogs, I looked at the throttle dip on descent start.
My copter reach 0.73g with throttle at 67% of hover(1220), or 60 units under.
Stronnag's copter reached 0.73g with throttle at 79% of hover(), or 100 units under.

But the thing is, on touchdown the throttle change looks almost identical in absolute numbers between my and stronnag's copter. I guess the althold controller works with absolute throttle values?